addons/mail/models/mail_notification.py

```diff
@@ -51,6 +54,32 @@ class MailNotification(models.Model):
          'Customer is required for inbox / email notification'),
     ]
 
+    # ------------------------------------------------------------
+    # COMPUTE
+    # ------------------------------------------------------------
+
+    @api.depends('mail_message_id.model', 'mail_message_id.res_id', 'res_partner_id')
+    def _compute_follower_id(self):
```

Comment:
Do we really need this ? I mean, a notification is linked to a message which is linked to a record, we can probably have simpler references than going through the notification ...